SayTheirNames/Source/Controller/Donations/DonationsMoreDetails/DonationsMoreDetailsController+Layout.swift

```diff
@@ -88,7 +88,36 @@ extension DonationsMoreDetailsController {
                     NSCollectionLayoutBoundarySupplementaryItem(layoutSize: titleViewSize,
                                                                 elementKind: DonationsMoreDetailsController.sectionTitleSupplementaryView,
                                                                 alignment: .top)
-
+                
+                section.boundarySupplementaryItems = [titleView]
+                
+                return section
+                
+            // Outcome section layout
+            case DonationSectionLayoutKind.outcome.rawValue:
+                // Item
+                let itemSize = NSCollectionLayoutSize(widthDimension: .fractionalWidth(1.0),
+                                                      heightDimension: FeatureFlags.dmdOutcomeSectionEnabled ?
+                                                        .estimated(200.0) : .absolute(0.1))
```

Comment:
I've looked a bit closer at the code and not sure I'm understanding it right. 
The section is there still, just having cells of height 0.1? It would make more sense to not add the section to the collection view altogether.

Reply:
Agreed. It's in the changes.